proof/refine/RISCV64/ArchCSpace_R.thy

```diff
@@ -274,6 +250,20 @@ end (* Arch_mdb_insert *)
 
 context Arch_mdb_insert_sib begin
 
+(* FIXME arch-split: move: AARCH64 has this in ArchCSpace1_R *)
+lemma not_untyped:
```

Comment:
Now that this chunk is finished, is it worth moving them before we merge? (I think all the non-AARCH64 theories have this)